source/projects/rush_hour.md

```diff
@@ -195,21 +204,21 @@ A Client has two attributes, an `identifier`, and a `rootUrl`.
 
 Create migrations to do the following:
 * Create the `Client` table with the necessary attributes
-* Create a migration to add a reference to the `Client` on the `PayloadRequest` table. This migration will establish the one-to-many relationship that `PayloadRequest`s and `Client`s have.
+* Create a migration to add a reference to the `Client` on the `Payload` table. This migration will establish the one-to-many relationship that `Payload`s and `Client`s have.
 
-Now that we have a place to store out client data, make sure you go into the models and establish the relationships between `PayloadRequest`, and `Client`, and you set up appropriate validations for the `Client`.
+Now that we have a place to store out client data, make sure you go into the models and establish the relationships between `Payload`, and `Client`, and you set up appropriate validations for the `Client`.
 
 Join the client table with its resources. For example:
 
 ```ruby
 class Client < ActiveRecord::Base
   #More Code Here
-  has_many :payload_requests
+  has_many :payloads
   #More Code Here
 end
 ```
 
-If your schema utilizes a join table to connect resources, make sure you remember to link the resources with `:through`. Take a look at [this documentation on active record associations](http://guides.rubyonrails.org/association_basics.html#the-has-many-through-association) to get it to work.
+If your schema utilizes a join table to connect resources, make sure you remember to link the resources with `:through`. For example, a `Client` may have many `request_types` through `payload_requests`. Take a look at [this documentation on active record associations](http://guides.rubyonrails.org/association_basics.html#the-has-many-through-association) to get it to work.
```

Comment:
Should this be that a `Client` may have many `request_types` through `payloads`?